### ISSUE TITLE
"explore video" pops up when the video is paused.

### ISSUE THREAD

**Issue by haikal-shah** [stamat/youtube-background#63]
Referring to the image below; when I paused the video player, the "explore video" tab popped up at the bottom of the screen. Is there a way to disable it?

Thank you for taking your time to read this.

![image](https://github.com/stamat/youtube-background/assets/112292898/d32ea936-69a1-4cc0-b5cf-6124441aaea2)

**Comment by stamat** [comment]
Hey bud @haikal-shah ✨ Unfortunately we cannot disable this since Sept 2018... as mentioned in this changelog https://developers.google.com/youtube/player_parameters#release_notes_08_23_2018

We do pass `rel=0` and `showinfo=0` query parameters, but like this changelog says, they are not effective anymore.

You can either zoom the iframe so the recommended videos will appear under the fold via css, or when the video is paused you can add a linear gradient form the bottom of the video to cover them lol, or you can upload the video via vimeo or host it by yourself as an mp4.

Basically this small package was intended to be used as a looping background (in this case the recommended videos will never appear since it loops. But the community requested extra player features, so that's why the extensive capabilities.

Hope this helps ☺️  Thanks for using this package, it means a lot!

**Comment by haikal-shah** [comment]
Hello @stamat

Thank you very much for taking the time to read and respond to the thread. As someone who's new to the programming world, I really appreciate the warm welcoming message for using the package, especially since it's my first issue thread haha.

You've earned a new follower! I will close the thread and hope to see you again 👋

**Comment by stamat** [comment]
@haikal-shah welcome to the programming world! My advice would be that you never give up! When things seem hard know that everything has a reason and that everything is solvable!

May our paths cross again! Best of luck friend! 👋 ✨
